<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1207</width>
    <height>860</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Task2 Создание ведомости</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>60</y>
      <width>611</width>
      <height>421</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>152</number>
    </attribute>
    <column>
     <property name="text">
      <string>ФИО</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Специальность</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Номер группы</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Средняя оценка за 
семестр</string>
     </property>
    </column>
   </widget>
   <widget class="QFrame" name="addFrame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>251</width>
      <height>491</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>101</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>ФИО</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="fioAdd_line">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>221</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>221</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Наименование специальности</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="specAdd_line">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>221</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>221</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Номер группы</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="groupAdd_line">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>221</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>241</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Оценки за предыдущий семестр:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>233</y>
       <width>66</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>ОАиП</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="oaipAdd_spinBox">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>229</y>
       <width>44</width>
       <height>26</height>
      </rect>
     </property>
     <property name="maximum">
      <number>10</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="maAdd_spinBox">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>266</y>
       <width>44</width>
       <height>26</height>
      </rect>
     </property>
     <property name="maximum">
      <number>10</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>270</y>
       <width>91</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Мат анализ</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="agilaAdd_spinBox">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>306</y>
       <width>44</width>
       <height>26</height>
      </rect>
     </property>
     <property name="maximum">
      <number>10</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>310</y>
       <width>91</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>АГиЛА</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="mlAdd_spinBox">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>346</y>
       <width>44</width>
       <height>26</height>
      </rect>
     </property>
     <property name="maximum">
      <number>10</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>350</y>
       <width>91</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Мат логика</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="hisoryAdd_spinBox">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>386</y>
       <width>44</width>
       <height>26</height>
      </rect>
     </property>
     <property name="maximum">
      <number>10</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>390</y>
       <width>91</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>История</string>
     </property>
    </widget>
    <widget class="QPushButton" name="AddButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>430</y>
       <width>201</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Добавить</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>141</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить студента</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>30</y>
      <width>141</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Студенты</string>
    </property>
   </widget>
   <widget class="QFrame" name="editFrame">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>60</y>
      <width>251</width>
      <height>581</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_75">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>54</y>
       <width>101</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>ФИО</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="fioEdit_line">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>74</y>
       <width>221</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_76">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>114</y>
       <width>221</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Наименование специальности</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="specEdit_line">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>134</y>
       <width>221</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_77">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>174</y>
       <width>221</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Номер группы</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="groupEdit_line">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>194</y>
       <width>221</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_78">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>234</y>
       <width>241</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Оценки за предыдущий семестр:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_79">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>267</y>
       <width>66</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>ОАиП</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="oaipEdit_spinBox">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>263</y>
       <width>44</width>
       <height>26</height>
      </rect>
     </property>
     <property name="maximum">
      <number>10</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="maEdit_spinBox">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>300</y>
       <width>44</width>
       <height>26</height>
      </rect>
     </property>
     <property name="maximum">
      <number>10</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_80">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>304</y>
       <width>91</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Мат анализ</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="agilaEdit_spinBox">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>340</y>
       <width>44</width>
       <height>26</height>
      </rect>
     </property>
     <property name="maximum">
      <number>10</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_81">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>344</y>
       <width>91</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>АГиЛА</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="mlEdit_spinBOx">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>380</y>
       <width>44</width>
       <height>26</height>
      </rect>
     </property>
     <property name="maximum">
      <number>10</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_82">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>384</y>
       <width>91</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Мат логика</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="historyEdit_spinBox">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>420</y>
       <width>44</width>
       <height>26</height>
      </rect>
     </property>
     <property name="maximum">
      <number>10</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_83">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>424</y>
       <width>91</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>История</string>
     </property>
    </widget>
    <widget class="QPushButton" name="delete_button">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>520</y>
       <width>201</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Удалить</string>
     </property>
    </widget>
    <widget class="QPushButton" name="edit_button">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>470</y>
       <width>201</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Изменить</string>
     </property>
    </widget>
    <widget class="QComboBox" name="studentSelect_comboBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>221</width>
       <height>25</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Выбрать студента</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QLabel" name="label_84">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>30</y>
      <width>141</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Изменить студента</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>490</y>
      <width>291</width>
      <height>151</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_85">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>121</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Выбрать группу</string>
     </property>
    </widget>
    <widget class="QComboBox" name="selectGroup_comboBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>270</width>
       <height>31</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Все группы</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="sort_button">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>270</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Отсортировать</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>490</y>
      <width>311</width>
      <height>151</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="openFile_button">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>291</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Открыть файл</string>
     </property>
    </widget>
    <widget class="QPushButton" name="saveFile_button">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>291</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Сохранить файл</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_3">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>650</y>
      <width>881</width>
      <height>161</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="find_button">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>281</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Найти</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_95">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>281</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Введите ФИО студента (или часть ФИО)</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>281</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QFrame" name="frame_4">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>10</y>
       <width>571</width>
       <height>141</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_96">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>10</y>
        <width>66</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Оценки:</string>
      </property>
     </widget>
     <widget class="QLabel" name="oaipLabel">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>30</y>
        <width>171</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="maLabel">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>50</y>
        <width>171</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="agilaLabel">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>70</y>
        <width>171</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="mlLabel">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>90</y>
        <width>171</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="historyLabel">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>110</y>
        <width>171</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="Group_Label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>291</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="Spec_Label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>291</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="Fio_Label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>291</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="AvMark_Label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>291</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>60</y>
        <width>151</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Студент не найден</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="_createVed_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>570</y>
      <width>251</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Создать новую ведомость</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>670</y>
      <width>231</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;При создании новой ведомости, &lt;/p&gt;&lt;p&gt;несохранённые даные будут &lt;/p&gt;&lt;p&gt;удалены&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <property name="indent">
     <number>-1</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1207</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
